lib/ice_cube/validations/day_of_week.rb

```diff
@@ -15,6 +15,8 @@ class Validation
       attr_reader :day, :occ
 
       def initialize(day, occ)
+        raise ArgumentError, "Invalid day_of_week occurrence: #{occ}" if occ == 0 || occ > 5 || occ < -5
```

Comment:
@seejohnrun - I'm curious why input validations like this weren't added to all of the validation classes. Was it a deliberate decision not to do this?